main.nf

```diff
@@ -156,7 +156,12 @@ workflow {
 
     if ( !params.skip_amr ) {
         AMRFINDER_PLUS( PROKKA.out.prokka_fna, PROKKA.out.prokka_faa, PROKKA.out.prokka_gff )
-        AMRFINDER_REPORT( AMRFINDER_PLUS.out.amrfinder_tsv, INTEGRATOR.out.mobilome_prokka_gff )
+	if ( params.user_genes ) {
+            user_gff = Channel.fromPath( params.prot_gff, checkIfExists: true )
+            AMRFINDER_REPORT( AMRFINDER_PLUS.out.amrfinder_tsv, INTEGRATOR.out.mobilome_prokka_gff, RENAME.out.map_file, user_gff )
+	} else {
+            user_gff = file('no_user_gff')
+            AMRFINDER_REPORT( AMRFINDER_PLUS.out.amrfinder_tsv, INTEGRATOR.out.mobilome_prokka_gff, RENAME.out.map_file, no_user_gff )
```

Comment:
```suggestion
        user_gff = file('no_user_gff')
	if ( params.user_genes ) {
            user_gff = file( params.prot_gff, checkIfExists: true )
	}
        AMRFINDER_REPORT(
            AMRFINDER_PLUS.out.amrfinder_tsv,
            INTEGRATOR.out.mobilome_prokka_gff,
            RENAME.out.map_file,
            user_gff
        )
```
Slightly easier to read IMO. Also, the user_gff file should be a file and not a channel. If it's a channel, it will run only once (which is OK for this version of the pipeline, but if we add multiple assemblies as input in the future, it will cause problems).